src/Restorer/PgSQL/Restorer.php

```diff
@@ -18,48 +19,37 @@ class Restorer extends AbstractRestorer
     public function startRestore(): AbstractRestorer
     {
         $dbParams = $this->connection->getParams();
-        $args = [
-            $this->binary,
-        ];
-
+        $command = new CommandLine($this->binary);
 
         if (isset($dbParams['host'])) {
-            $args[] = '-h';
-            $args[] = $dbParams['host'];
+            $command->addArg('-h', $dbParams['host']);
         }
-
         if (isset($dbParams['user'])) {
-            $args[] = '-U';
-            $args[] = $dbParams['user'];
+            $command->addArg('-U', $dbParams['user']);
         }
-
         if (isset($dbParams['port'])) {
-            $args[] = '-p';
-            $args[] = $dbParams['port'];
+            $command->addArg('-p', $dbParams['port']);
         }
 
-        $args[] = '-w';
-        $args[] = '--clean';
-        $args[] = '-d';
-        $args[] = $dbParams['dbname'];
-        $args[] = '-j';
-        $args[] = 2;
-        $args[] = '--if-exists';
-        $args[] = '--disable-triggers';
-        $args[] = $this->backupFilename;
+        $command->addArg('-w');
+        $command->addArg('--clean');
+        $command->addArg('-j', '2');
+        $command->addArg('--if-exists');
+        $command->addArg('--disable-triggers');
```

Comment:
Why do you not consider these options as "extraOptions" ?

Reply:
Seems legit to me, but I just wanted to know if this was omitted or intentional

Reply:
It was not intentional.

Because I considered that all options that was not related to the database connection, table to exclude, verbosity and output or input file was _extra options_. But we can discuss about this point.

Reply:
Fixed as well.